packages/core/src/components/TextField/TextField.tsx

```diff
@@ -1,7 +1,11 @@
+import { InfoIcon, InfoOutlineIcon } from '@medly-components/icons';
+import { defaultTheme } from '@medly-components/theme';
 import { useCombinedRefs, WithStyle } from '@medly-components/utils';
 import React, { FC, FocusEvent, FormEvent, useCallback, useEffect, useMemo, useState } from 'react';
+import Popover from '../Popover';
 import getMaskedValue from './getMaskedValue';
 import * as Styled from './Styled';
+import { HelperTextForTooltip } from './Styled';
```

Comment:
@charp2  Here the `./Styled` import is duplicate (code smell). Since it's already present on line number 7, we can write `<Styled.HelperTextForTooltip/>` in the tsx part of the component.

Reply:
Ah yes, good catch!